en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc

```diff
@@ -330,7 +330,7 @@ The *Account* page appears, displaying a linking request from Anypoint Platform.
 . In the *Access Token* section, click *Authenticate in Anypoint*. +
 The Anypoint Platform login page appears.
 
-. Log in to Anypoint Platform.
+. Log in to Anypoint Platform. Ensure that the user logging into Anypoint Platform has the right https://docs.mulesoft.com/api-manager/2.x/access-client-app-id-task[permissions to create a client app].
```

Comment:
I believe you can specify the version thusly:
`xref:2.x@api-manager::access-client-app-id-task[]`

Reply:
Thank you!